backend/analytics_server/mhq/service/code/repository_service.py

```diff
@@ -22,6 +22,9 @@ def __init__(
     def get_team_repos(self, team: Team) -> List[OrgRepo]:
         return self._code_repo_service.get_team_repos(team_id=str(team.id))
 
+    def get_team_repos_by_team_id(self, team: Team) -> List[TeamRepos]:
+        return self._code_repo_service.get_team_repos_by_team_id(team_id=str(team.id))
+
```

Comment:
`get_team_repos_by_team_id` we should update the func name as we are fetching the entity by passing the team object and not just the id.
How about get_team_repos_mapping ?

Reply:
Yeah makes sense. `get_team_repos_mapping` is a little misleading since it doesn't really return a map. How about `get_team_repos_by_team`?